glide.yaml

```diff
@@ -87,13 +87,13 @@ import:
 
 # openshift second
 - package: github.com/openshift/api
-  version: master
+  version: release-4.2
```

Comment:
This feels like something that should happen automatically when we fork an origin branch off master.  Is there a plan to make that happen, or is it up to whoever makes the first post-fork vendor bump to do this?

Reply:
Yeah, I think this was missed upon branching.